packages/react-urql/examples/1-getting-started/package.json

```diff
@@ -15,11 +15,11 @@
   },
   "dependencies": {
     "@urql/devtools": "latest",
-    "graphql": "^14.6.0",
+    "graphql": "^15.0.0",
```

Comment:
Heads up, I had some issues with breaking changes for this release - if you've tested it on your end then it's probably fine!